R/openmp-utils.R

```diff
@@ -1,12 +1,12 @@
-setDTthreads = function(threads=NULL, restore_after_fork=NULL, percent=NULL) {
+setDTthreads = function(threads=NULL, restore_after_fork=NULL, percent=NULL, th_iters=NULL) {
```

Comment:
not a fan of the argument name... maybe `par_threshold` or `min_iter` or `onset_size`